package/MDAnalysis/make_bundle.py

```diff
@@ -3,24 +3,26 @@
 
 from .core.AtomGroup import Universe
 
-def make_bundle(*args, **kwargs):
+def make_bundle(simulations, topology=None, univ_kwargs={}, names=None, 
+                data={}, auxs={}, aux_kwargs={}):
```

Comment:
Having dicts as default arguments is not a good idea. They are mutable leading to weird problems when you create more then one bundle. Rather make then `None` and convert them to empty dicts in the function body.